code/modules/mob/spells.dm

```diff
@@ -1,4 +1,7 @@
 /mob/proc/add_spell(var/spell/spell_to_add, var/spell_base = "wiz_spell_ready", var/master_type = /obj/screen/movable/spell_master)
+	if(ispath(spell_to_add, /spell))
```

Comment:
It'll return false if it's an instance or anything that isn't a path (just like istype will return false if fed a null or a path)